test/lexers/javascript-lexer.test.js

```diff
@@ -109,15 +100,12 @@ describe('JavascriptLexer', () => {
     ])
     done()
   })
-
-  describe('supports the acorn-stage3 plugin', () => {
-
-    it('supports dynamic imports', (done) => {
```

Comment:
Same, please don't remove this test, it was added to support some users and we need to know it still works.